open-api/rest-catalog-open-api.yaml

```diff
@@ -770,6 +770,52 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/{prefix}/namespaces/{namespace}/tables/{table}/metrics:
+    parameters:
+      - $ref: '#/components/parameters/prefix'
+      - $ref: '#/components/parameters/namespace'
+      - $ref: '#/components/parameters/table'
+
+    post:
+      tags:
+        - Catalog API
+      summary: Send a scan report to this endpoint to be processed by the backend
+      description:
+        Send a scan report to this endpoint to be processed by the backend
```

Comment:
Summary and description are identical. Should we remove one?

Reply:
done